### ISSUE TITLE
Голосования

### ISSUE THREAD

**Issue by Archemagus** [animusdev/Wonderwork#17]
Из лобби нельзя голосовать, так же говорят, за гостов тоже

**Comment by HAKCA** [comment]
Это не баг, а фича. Но данный запрет распространяется на предусмотренные игрой голосования? Или же ещё и на кастомные?

**Comment by Cyrpet** [comment]
На 'кастомное', созданное во время лобби, никто не смог проголосовать.

**Comment by Archemagus** [comment]
На все к сожалению. И я считаю, что нужно дать возможность голосовать всем. Но на кастомные - точно.

**Comment by Archemagus** [comment]
Да еще одна проблема, довольно важная.
При голосовании за крю трансфер не зависимо от результата шаттл не вызывается.

**Comment by OdSt** [comment]
Да, это все защита от рестартоебов. Решайте сами - нужно или нет.

**Comment by ucnaHez** [comment]
А давайте вообще выпилим эти убогие голосования на ристорт и шатол. Хочешь шатол - иди и вызови, мать вашу. Даже ветер это понял, не должны соснувшие влиять на игру, не должны люди иметь возможность через ООС влиять на игру, нельзя давать людям возможность вызвать шатол обойдя все, что там сделал укрепившийся и спиздивший все консоли инж, просто использовав магический аут-оф-чарактер верб.

Если вещи идут совсем серьезно и шатол нужен, а взывать никак нельзя - так педали есть, пусть ребутают, иначе - вызывайте ингейм.
